src/liquid/components/ld-notification/ld-notification.css

```
@@ -0,0 +1,159 @@
+.ld-notification {
+  position: fixed;
+  z-index: 2147483647; /* Highest possible */
+  display: flex;
+  justify-content: center;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  min-width: 20rem;
+  font: var(--ld-typo-body-m);
+  pointer-events: none;
+}
+
+@keyframes ld-notification-fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes ld-notification-fade-out {
+  from {
+    transform: none;
+    opacity: 1;
+  }
+  to {
+    transform: none;
+    opacity: 0;
+  }
+}
+
+.ld-notification__item {
+  pointer-events: auto;
+  position: absolute;
+  transform-origin: center;
+  box-shadow: var(--ld-shadow-sticky);
+  width: fit-content;
+  min-width: 19rem;
+  max-width: calc(90% - 1rem);
+  max-height: 80vh;
+  min-height: 2.5rem;
```

```suggestion
  min-height: var(--ld-sp-40);
```

Same as for the dismiss button.